web/src/constants/layers.ts

```diff
@@ -0,0 +1,63 @@
+import { LayerProps } from 'react-map-gl'
+
+import { mapMarker, PoiFeature } from 'api-client'
+import { ThemeType } from 'build-configs/ThemeType'
+
+const textOffsetY = 1.25
+
+const groupCount = 50
+const circleRadiusSmall = 20
+const circleRadiusLarge = 30
+const fontSizeSmall = 12
+const fontSizeLarge = 16
+
+export const clusterLayer = (theme: ThemeType): LayerProps => ({
+  id: 'clusters',
+  type: 'circle',
+  source: 'point',
+  filter: ['has', 'point_count'],
+  paint: {
+    'circle-color': [
+      'step',
+      ['get', 'point_count'],
+      `${theme.colors.themeColor}`,
+      groupCount,
+      `${theme.colors.themeColor}`,
```

Comment:
Any specific need for the template string?
```suggestion
      theme.colors.themeColor,
      groupCount,
      theme.colors.themeColor,
```